docker/run_tests.sh

```diff
@@ -19,16 +19,14 @@ set -v -e -x
 echo ">>> pytest"
 # Set up environment variables
 
-LOCALSTACK_S3_URL=http://localstack-s3:5000
-PUBSUB_URL=http://pubsub:5010
```

Comment:
These were hardcoded. I unhardcoded them and now they pull from variables in the environment.